lib/options.js

```diff
@@ -108,6 +108,12 @@ module.exports = optionator({
         description: "Report errors only"
     },
     {
+        option: "max-warnings",
+        type: "Number",
+        default: "-1",
+        description: "Maximum warning count before eslint exits with an error (or -1 for no limit)"
```

Comment:
Can you shorten this description? It's a bit too long relative to the others (we don't want it to wrap on consoles).